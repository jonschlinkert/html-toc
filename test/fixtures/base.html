<!DOCTYPE html>
<html lang="en">

<head>
  <meta charset="UTF-8">
  <title>Document</title>
</head>

<body>
  <div id="toc"></div>
  <h3>foo</h3>
  <h4>bar</h4>
</body>

</html>
