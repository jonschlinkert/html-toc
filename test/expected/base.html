<!DOCTYPE html>
<html lang="en">

<head>
  <meta charset="UTF-8">
  <title>Document</title>
</head>

<body>
  <div id="toc"><ul class="nav sidenav"><li><a href="#foo">foo</a><ul class="nav"><li><a href="#foo-bar">bar</a></li></ul></li></ul></div>
  <h3 id="foo">foo<a href="#foo" name="foo" class="anchor">
  <span class="anchor-target" id="foo"></span>
  <span class="glyphicon glyphicon-link"></span>
</a></h3>
  <h4 id="foo-bar">bar<a href="#foo-bar" name="foo-bar" class="anchor">
  <span class="anchor-target" id="foo-bar"></span>
  <span class="glyphicon glyphicon-link"></span>
</a></h4>
</body>

</html>
